src/examples/source/SmartPeakGUI.cpp

```diff
@@ -178,8 +178,8 @@ int main(int argc, char** argv)
   auto spectrum_explorer_window_ = std::make_shared<ExplorerWidget>("SpectrumExplorerWindow", "Scans", &event_dispatcher);
   auto sequence_main_window_ = std::make_shared<SequenceTableWidget>("SequenceMainWindow", "Sequence",
                                                                       &session_handler_, &application_handler_.sequenceHandler_);
-  auto transitions_main_window_ = std::make_shared<GenericTableWidget>("TransitionsMainWindow", "Transitions");
-  auto spectrum_main_window_ = std::make_shared<GenericTableWidget>("SpectrumMainWindow", "Scans");
+  auto transitions_main_window_ = std::make_shared<GenericTableWidget>("TransitionsMainWindow", "Transitions Table");
```

Comment:
A better idea is to substitute all the window/widget string IDs ("TransitionsMainWindow", "Transitions Table", ..) with enum classes to make sure they are unified across all other classes/functions.

Reply:
Indeed the thing here is because we have 2 window with the same name (Scans/Scans), (Transitions/Transitions) then the docker mechanism was lost and displays these windows at the wrong place.
So we have to have unique name across the application.
However an enum looks to me a very static way to solve this issue. It's possible that one day we want to have multiple instance of the same type of window, very dynamical (could be "chromatogram 1", "chromatogram 2" ...) and using an enum may be a problem there. perhaps a warning could be done instead. with some check in the main gui.

Reply:
We would definitely want a dynamic solution to ensure unique names for each widget in the future once we introduce the ability to copy and split windows.